packages/i18n/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "env": {
+    "es2021": true,
+    "node": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
```

Comment:
You don't need this 🙂. [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier) serves only to turn off ESLint rules that conflict with formatters such as Prettier... but neither `"eslint:recommended"` nor `"plugin:@typescript-eslint/recommended"` enable any like that anymore.